<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogAcaProgram</class>
 <widget class="QDialog" name="dialogAcaProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbAcaContent">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>761</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Contents</string>
   </property>
   <widget class="QTableView" name="tvAcaProgram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushBtnAcaProgram">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>361</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dynamic Button</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>341</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QRadioButton" name="rdbtnAddAcaProgram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add program</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnUpdateAcaProgram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update program</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtnDeleteAcaProgram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete program</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAcaEditData">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Data</string>
   </property>
   <widget class="QLabel" name="lbProgramCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbProgramname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTuition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuition</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAcaStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProgramCode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtProgramName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdAcaStatusOpen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdAcaStatusClose">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsbProgramTuition">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
